app/src/test/java/com/github/globant/githubsubscribers/subscribersdetail/SubscriberDetailPresenterTest.java

```diff
@@ -0,0 +1,225 @@
+
+        view.showSubscriberDetails(user);
+    }
+
+    /**
+     * Unit Test to try when a null answer is given from the User Info service
+     */
+    @Test
+    public void getNullUserInfoRequestFromInteractor() {
+
+        // Given a type error with fake error message
+        ErrorMessagesHelper.TypeError typeError = ErrorMessagesHelper.TypeError.BAD_ANSWER;
+        String errorMessage = Constants.EXCEPTION_ERROR + ": " + typeError.toString();
+
+        // When detail presenter is called to get a User information
+        presenter.getUser(USER_NAME_PARAM);
+
+        //Then User information is loaded from Interactor and the callback is captured
+        verify(interactor).getUserData(eq(USER_NAME_PARAM), OnFinishedListenerCallBack.capture());
+
+        OnFinishedListenerCallBack.getValue().onFailureUser(errorMessage, typeError);
+
+        int messageId = ErrorMessagesHelper.getMessage(typeError);
+        view.showUserError(messageId);
+    }
+
+    /**
+     * Unit Test to try when a Not Connection answer is given from the User Info service
+     */
+    @Test
+    public void getNotConnectionUserInfoRequestFromInteractor() {
+
+        // Given a type error with fake error message
+        ErrorMessagesHelper.TypeError typeError = ErrorMessagesHelper.TypeError.NO_CONNECTION;
+        String errorMessage = Constants.EXCEPTION_ERROR + ": " + typeError.toString();
+
+        // When detail presenter is called to get a User information
+        presenter.getUser(USER_NAME_PARAM);
+
+        //Then User information is loaded from Interactor and the callback is captured
+        verify(interactor).getUserData(eq(USER_NAME_PARAM), OnFinishedListenerCallBack.capture());
+
+        OnFinishedListenerCallBack.getValue().onFailureUser(errorMessage, typeError);
+
+        int messageId = ErrorMessagesHelper.getMessage(typeError);
+        view.showUserError(messageId);
+    }
+
+    /**
+     * Unit Test to try when a successful answer is given from the Repositories service
+     */
+    @Test
+    public void getRepositoriesListFromInteractorAndLoadIntoView() {
+
+        // Given an initialized Repository object with fake data and a list is created
+        Repository repo1 = new Repository(id_repo1, name_repo1, fullName_repo1, html_url_repo1);
+        Repository repo2 = new Repository(id_repo2, name_repo2, fullName_repo2, html_url_repo2);
+        ArrayList<Repository> repoList = new ArrayList<>();
+        repoList.add(repo1);
+        repoList.add(repo2);
+
+        // When detail presenter is called to get a Repositories list
+        presenter.getRepositoryList(USER_NAME_PARAM);
+
+        //Then Repositories list is loaded from Interactor and the callback is captured
+        verify(interactor).getUserRepositoryData(eq(USER_NAME_PARAM), OnFinishedListenerCallBack.capture());
+
+        OnFinishedListenerCallBack.getValue().onFinishedRepository(repoList);
+
+        view.showSubscriberUserRepositories(repoList);
```

Comment:
Yo need to call `verify(view).method(param)`. Please refer to https://codelabs.developers.google.com/codelabs/android-testing/index.html?index=..%2F..%2Findex#5